src/ReactotronConfig.js

```diff
@@ -5,16 +5,19 @@ import { NativeModules } from 'react-native';
 import { reactotronRedux } from 'reactotron-redux';
 
 // uncomment the following lines to test on a device
-// let scriptHostname;
-// if (__DEV__) {
-//   const scriptURL = NativeModules.SourceCode.scriptURL;
-//   scriptHostname = scriptURL.split('://')[1].split(':')[0];
-// }
 
-Reactotron.configure(/*{ host: scriptHostname }*/) // controls connection & communication settings
+if (__DEV__) {
```

Comment:
Surrounded whole Reactotron related code in `__DEV__` so that nothing else gets executed beyond dev environment.